web/src/helpers/utils.ts

```diff
@@ -134,3 +134,9 @@ export const parseHTMLToRawText = (htmlStr: string): string => {
   const text = tempEl.innerText;
   return text;
 };
+
+export const setCaretPosition = (pos: number) => {
```

Comment:
It will be better to move this function to [`Editor.tsx`](https://github.com/usememos/memos/blob/main/web/src/components/Editor/Editor.tsx#L5).